content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+
+| Path | Purpose |
+| :--- | :--- |
+| ```/opt/alfresco```     | Binaries |
+| ```/etc/opt/alfresco``` | Configuration |
+| ```/var/opt/alfresco``` | Data |
+| ```/var/log/alfresco``` | Logs |
+
+## Service Configuration
+
+The following systemd services are installed and can be used to stop and start Alfresco components:
+
+| Service Name | Purpose |
+| :--- | :--- |
+| ```activemq.service``` | ActiveMQ Service |
+| ```postgresql-<version>.service``` | Postgresql DB Service (where `<version>` is 11 for ACS 6.2.N and 13 for ACS 7.x) |
+| ```nginx.service``` | Nginx Service |
+| ```tomcat.service``` | Tomcat Service |
+| ```alfresco-content.service``` | Alfresco Content Service |
+| ```alfresco-search.service``` | Alfresco Search Service |
+| ```alfresco-shared-fs.service``` | Alfresco Shared File Store Controller Service |
+| ```alfresco-sync.service``` | Alfresco Sync Service |
+| ```alfresco-tengine-aio.service``` | Alfresco AIO Transform Core Engine |
+| ```alfresco-transform-router.service``` | Alfresco Transformation Router Service |
+
+## TCP Port Configuration
+
+Several roles setup services that listen on TCP ports and several roles wait for TCP ports to be listening before continuing execution (indicated by `Yes` in the "Required For Installation" column). The table below shows the communication paths and port numbers used.
+
+| Target Host | Target Port | Source Hosts | Required For Installation |
+| :--- | :--- | :--- | :--- |
+| activemq | 61616 | repository, syncservice, transformers | Yes |
+| database | 5432 | repository, syncservice | Yes |
+| repository | 8080 | nginx, search, syncservice | Yes |
+| search | 8983 | repository | No |
+| transformers (aio t-engine) | 8090 | repository | No |
+| syncservice | 9090 | nginx | No |
+| adw | 80 | nginx | No |
+| nginx | 80 | `<client-ips>` | No |
+| nginx | 443 | `<client-ips>` | No |
+
+> NOTE: The transformers host will also contain the transform router process running on port 8095 and the shared file system process running on 8099 but communication between these components remains local.
+
+# Configure your installation
+
+By default, without any configuration applied, the playbook will install a limited trial of the Enterprise version of Alfresco Content Services 7.x that goes into read-only mode after 2 days. If you'd like to try Alfresco Content Services for a longer period, request the 30-day [Download Trial](https://www.alfresco.com/platform/content-services-ecm/trial/download).
+
+The sections below describe how you can configure your installation before running the playbook.
+
+## License
+
+If you have a valid license place your `*.lic` file in the `configuration_files/licenses` folder before running the playbook.
+
+>NOTE: You can also [upload a license](https://docs.alfresco.com/6.2/tasks/at-adminconsole-license.html) via the Admin Console once the system is running.
```

Comment:
Incorrect link and format: https://docs.alfresco.com/6.2/tasks/at-adminconsole-license.html